client/src/components/brushableHistogram/index.tsx

```diff
@@ -258,6 +276,29 @@ class HistogramBrush extends React.PureComponent<BrushableHistogramProps> {
     const summary = column.summarizeContinuous();
     const range = [summary.min, summary.max];
 
+    // seve: if the anno matrix is not a view and it is a single value, add remove it from histograms and send it to the dataset drawer
+    // NOTE: this also includes embedding views, so if the default embedding subsets to a view and there is a continuous value it will not be added to the dataset drawer
```

Comment:
Correct

Reply:
```suggestion
    // NOTE: this also includes embedding views, so if the default embedding subsets to a view and there is a single continuous value for a field, it will not be added to the dataset drawer
```